dataRetriever/resolvers/transactionResolver.go

```diff
@@ -150,7 +150,7 @@ func (txRes *TxResolver) resolveTxRequestByHashArray(hashesBuff []byte, pid p2p.
 		tx, errTemp := txRes.fetchTxAsByteSlice(hash)
 		if errTemp != nil {
 			errFetch = errTemp
-			err = fmt.Errorf("%w for hash %s", errFetch, logger.DisplayByteSlice(hash))
+			errFetch = fmt.Errorf("%w for hash %s", errFetch, logger.DisplayByteSlice(hash))
```

Comment:
do one line errFetch = fmt.Errorf("%w for hash %s", errTemp, logger.DisplayByteSlice(hash))

Reply:
done